More edits for catagories, channel logos

jkemp814 opened the pull request:
Will be performing many edits. Figuring out a good work flow. Any suggestions would be greatly appreciated. If there is an area of interest on any of the playlists that needs attention, just let me know or issue assignment.

jkemp814 committed:
More edits for catagories, channel logos

jkemp814 committed:
Edited bon appétit to match TV logo and magazine

jkemp814 committed:
Remove wite space in channel name

jkemp814 committed:
Matched channel logos for Bloomberg

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -424,139 +424,139 @@ https://bloodydisgusting-roku-us.cinedigm.com/midroll/amagi_hls_data_xumo-host-r
+#EXTINF:-1 tvg-id="BloombergTV.us" tvg-name="Bloomberg TV" tvg-country="US" tvg-language="" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg TV
 https://bloomberg-bloomberg-1-eu.rakuten.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="Bloomberg.us" tvg-name="Bloomberg" tvg-country="US" tvg-language="" tvg-logo="https://od.lk/s/MF8yMzE5OTIyMjBf/Redbox_512x512.png" group-title="Business",Bloomberg
+#EXTINF:-1 tvg-id="BloombergTV.us" tvg-name="Bloomberg" tvg-country="US" tvg-language="" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg TV
```
Inconsistency in the tags

jkemp814 replied:
Fixed

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -424,139 +424,139 @@ https://bloodydisgusting-roku-us.cinedigm.com/midroll/amagi_hls_data_xumo-host-r
+#EXTINF:-1 tvg-id="BloombergTV.us" tvg-name="Bloomberg" tvg-country="US" tvg-language="" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg TV
 https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxbloomberg/CDN/playlist.m3u8
-#EXTINF:-1 tvg-id="BloombergEN.us" tvg-name="Bloomberg (EN)" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/fhcaNlc.png" group-title="Business",Bloomberg (EN)
+#EXTINF:-1 tvg-id="BloombergTV.us" tvg-name="Bloomberg" tvg-country="US" tvg-language="" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg TV
```
Inconsistency in the tags

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -424,139 +424,139 @@ https://bloodydisgusting-roku-us.cinedigm.com/midroll/amagi_hls_data_xumo-host-r
+#EXTINF:-1 tvg-id="BloombergQuicktakePortugal.us" tvg-name="Bloomberg Quicktake (Portugal)" tvg-country="US" tvg-language="" tvg-logo="https://www.adweek.com/wp-content/uploads/2019/12/quicktake-bloomberg-tictoc-CONTENT-2019-640x360.jpg" group-title="General",Bloomberg Quicktake (Portugal)
 https://bloomberg-quicktake-1-pt.samsung.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="BloombergTelevisionMexico.us" tvg-name="Bloomberg Television (Mexico)" tvg-country="US" tvg-language="" tvg-logo="" group-title="Business",Bloomberg Television (Mexico)
+#EXTINF:-1 tvg-id="BloombergTVMexico.us" tvg-name="Bloomberg Television (Mexico)" tvg-country="US" tvg-language="" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg TV (Mexico)
```
Inconsistency with the tags above.

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -424,139 +424,139 @@ https://bloodydisgusting-roku-us.cinedigm.com/midroll/amagi_hls_data_xumo-host-r
 #EXTINF:-1 tvg-id="Bluegrass4U.us" tvg-name="Bluegrass 4U" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/nm3N3aw.png" group-title="",Bluegrass 4U
 https://59d39900ebfb8.streamlock.net/blugrassmusic/blugrassmusic/playlist.m3u8
-#EXTINF:-1 tvg-id="BonAppetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="",Bon Appetit
+#EXTINF:-1 tvg-id="bonappétit.us" tvg-name="bon appétit" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="",bon appétit
```
Inconsistency with the tags below

jkemp814 replied:
This is what I have: #EXTINF:-1 tvg-id="bonappétit.us" tvg-name="bon appétit"

I have been have problems with my commit push through. That's how their channel logo looks

jkemp814 replied:
I thought the tvg-id could have spaces

LaneSh4d0w replied:
TVG-id doesn't have spaces indeed. said this because the end of that line has got a different name that the one you gave below.

jkemp814 replied:
I have four of the channels, and they all look the same except the (1080p). That's allowed right?

LaneSh4d0w replied:
That's OK.

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -2034,23 +2034,23 @@ https://vod365-ketchuptv-1.roku.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="KidoodleTV.us" tvg-name="Kidoodle TV" tvg-country="CA" tvg-language="CA" tvg-logo="https://i.imgur.com/KHkerRD.png" group-title="Kids",Kidoodle TV (720p)
 https://kidoodletv-kdtv-1-ca.samsung.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="KidoodleTV.us" tvg-name="Kidoodle_TV" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/aP8Wz2G.png" group-title="",Kidoodle TV
+#EXTINF:-1 tvg-id="KidoodleTV.us" tvg-name="Kidoodle_TV" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/aP8Wz2G.png" group-title="Kids",Kidoodle TV
```
Inconsistency with the tags above

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -2034,23 +2034,23 @@ https://vod365-ketchuptv-1.roku.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="KidoodleTV.us" tvg-name="Kidoodle TV" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/MaFtbM5.png" group-title="Kids",Kidoodle TV
 http://kidoodletv-kdtv-3.vizio.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="KiDoodleTV.us" tvg-name="KiDoodle TV" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/MaFtbM5.png" group-title="",KiDoodle TV
+#EXTINF:-1 tvg-id="KiDoodleTV.us" tvg-name="KiDoodle TV" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/MaFtbM5.png" group-title="Kids",KiDoodle TV
```
Inconsistency with the tags below and above.

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -2034,23 +2034,23 @@ https://vod365-ketchuptv-1.roku.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="KidGamerTV.us" tvg-name="Kid Gamer TV" tvg-country="US" tvg-language="" tvg-logo="" group-title="Kids",Kid Gamer TV
 https://studio71-vizio.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="Kidoodle.us" tvg-name="Kidoodle" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/mMAO3P6.jpg" group-title="",Kidoodle (1080p)
+#EXTINF:-1 tvg-id="Kidoodle.us" tvg-name="Kidoodle" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/mMAO3P6.jpg" group-title="Kids",Kidoodle (1080p)
```
Inconsistency with the tags below

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -2034,23 +2034,23 @@ https://vod365-ketchuptv-1.roku.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="Kidoodle.us" tvg-name="Kidoodle" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/mMAO3P6.jpg" group-title="Kids",Kidoodle (1080p)
 http://kidoodletv-kdtv-3.roku.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="Kidoodle.us" tvg-name="Kidoodle" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/MaFtbM5.png" group-title="",Kidoodle (720p)
+#EXTINF:-1 tvg-id="Kidoodle.us" tvg-name="Kidoodle" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/MaFtbM5.png" group-title="Kids",Kidoodle (720p)
```
Inconsistency with the tags below

jkemp814 committed:
Matched channel logos

jkemp814 committed:
Fixed Bloomberg tag for consistency

jkemp814 committed:
Added group title tag

jkemp814 committed:
Fixed Kidoodle

jkemp814 committed:
Fixed Caps on Kidoodle TV

jkemp814 committed:
Added more channel logos

jkemp814 committed:
Added macthing logo to Docurama

Dum4G committed:
Merge branch 'master' into master

Dum4G commented on channels/ae.m3u:
```diff
@@ -17,9 +17,9 @@ https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m
+#EXTINF:-1 tvg-id="AlAan.ae" tvg-name="Al Aan" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="Kids",Al Aan (720p) [Not 24/7]
 https://iptv--iptv.repl.co/streamlink?url=https://www.dailymotion.com/video/x74wje5
-#EXTINF:-1 tvg-id="AlAan.ae" tvg-name="Al Aan" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="",Al Aan (720p)
+#EXTINF:-1 tvg-id="AlAan.ae" tvg-name="Al Aan" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="Kids",Al Aan (720p)
```
![Uploading image.png…]()
Don't assume genres by momentary programming

jkemp814 replied:
Okay. I'll research some more. Those speakers can go for hours.

Dum4G committed:
Merge branch 'master' into master

jkemp814 committed:
Changed Al Aan group-title tag to culture

jkemp814 committed:
Merge branch 'master' of https://github.com/jkemp814/iptv

Dum4G commented on channels/ae.m3u:
```diff
@@ -17,9 +17,9 @@ https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m
 #EXTINF:-1 tvg-id="AjmanHD.ae" tvg-name="Ajman HD" tvg-country="AE" tvg-language="Arabic" tvg-logo="" group-title="",Ajman HD
 http://149.100.19.226:8001/play/AJMAN_HD
-#EXTINF:-1 tvg-id="AlAan.ae" tvg-name="Al Aan" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="Kids",Al Aan (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="AlAan.ae" tvg-name="Al Aan" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="Culture",Al Aan (720p) [Not 24/7]
```
More like General broadcast (Lifestyle and News)
https://www.alaan.tv/about-us

jkemp814 replied:
Okay. When I went to their website, I have seen they cater to young females in their culture. But do have other programming, including getting interrupted for state speeches. I changed it to General.

BellezaEmporium committed:
Merge branch 'master' into master

jkemp814 committed:
Various edits, gropu titles, names, logos

jkemp814 committed:
Changed  Al Aan  group-title tag to General

Dum4G committed:
Merge branch 'master' into master

jkemp814 committed:
More edits

jkemp814 committed:
Merge branch 'master' of https://github.com/jkemp814/iptv

jkemp814 committed:
Corrected logos on Mytime to be consistent

jkemp814 committed:
Edited News 12 New York, and NewsMax

jkemp814 committed:
Re-edited Newsmax

BellezaEmporium committed:
Merge branch 'master' into master

jkemp814 committed:
Edited logos for Newsy

jkemp814 committed:
Merge branch 'master' of https://github.com/jkemp814/iptv

jkemp814 committed:
Edited nosey, names, langauge, logos

Dum4G commented on channels/us.m3u:
```diff
@@ -261,17 +261,17 @@ http://45.6.4.154/americatuc/vivo.m3u8?PlaylistM3UCL
 #EXTINF:-1 tvg-id="AmericanHorrors.us" tvg-name="American Horrors" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/JLkzt7v.jpg" group-title="Movies",American Horrors (480p)
 http://170.178.189.66:1935/live/Stream1/playlist.m3u8
-#EXTINF:-1 tvg-id="AmericasVoice.us" tvg-name="Americas Voice" tvg-country="US" tvg-language="English" tvg-logo="" group-title="",Americas Voice
+#EXTINF:-1 tvg-id="AmericasVoice.us" tvg-name="America's Voice" tvg-country="US" tvg-language="English" tvg-logo="" group-title="https://is2-ssl.mzstatic.com/image/thumb/Purple113/v4/cb/8e/31/cb8e3181-4a9d-4fc3-b84f-f363026b32c8/source/512x512bb.jpg" group-title="News",America's Voice
```
Wrong tags

LaneSh4d0w replied:
America's Voice, Real America's Voice etc etc... are news channels.

https://americasvoice.news/
https://americasvoice.org/

jkemp814 replied:
Corrected.

jkemp814 committed:
Edited Orange TV and OAN

Dum4G commented:
General advise: treat tvg logos as a favicon - it's something that is always shown as a small element and at the same time we can have a lot of them. Imaging a strength that we add when a player has to download and resize each picture in such a big playlist with every update. I would recommend to not exceed 300p height

jkemp814 committed:
Corrected wrong character usage in tag

LaneSh4d0w commented:
Not the change we've expected.

Look at the end of the line for "America's Voice".

jkemp814 committed:
Corrected  America's Voice

jkemp814 committed:
Reverted the hyphen to match channel name

jkemp814 commented:
Understood on the logo pixel size and download source.

Dum4G commented:
Also prefer square versions over long rectangular ones if possible

jkemp814 committed:
Merge branch 'master' into master

jkemp814 committed:
The various edits continue

jkemp814 committed:
Many edits

jkemp814 committed:
Corrected All Weddings at line 226

jkemp814 committed:
Correct error on line 5642

jkemp814 committed:
Added group-title tag for Young Hollywood

jkemp814 committed:
Edited some Spanish channels (verified)

jkemp814 committed:
Corrected quotes on line 797

Dum4G commented on channels/us.m3u:
```diff
@@ -2352,7 +2352,7 @@ http://bamus-eng-roku.amagi.tv/playlist404p.m3u8
 #EXTINF:-1 tvg-id="LoveNature4K.us" tvg-name="Love Nature 4K" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/AzOvgkV.png" group-title="Outdoor",Love Nature 4K
 https://d27r4t30huw0iy.cloudfront.net/playlist.m3u8
-#EXTINF:-1 tvg-id="LoveNatureEspanol.us" tvg-name="Love Nature Espanol" tvg-country="US" tvg-language="Spanish" tvg-logo="https://image.xumo.com/v1/channels/channel/9999336/800x800.png?type=smartCast_channelTile" group-title="",Love Nature Español
+#EXTINF:-1 tvg-id="LoveNatureEspañol.us" tvg-name="Love Nature Español" tvg-country="US" tvg-language="Spanish" tvg-logo="https://image.xumo.com/v1/channels/channel/9999336/800x800.png?type=smartCast_channelTile" group-title="",Love Nature Español
```
We don't use special symbols in tvg-ids

jkemp814 replied:
Okay.

jkemp814 replied:
Made adjustments

jkemp814 committed:
Removed special symbols in tvg-ids

jkemp814 committed:
Additional edits

Dum4G committed:
Merge branch 'master' into master

jkemp814 committed:
Added missing logos

jkemp814 committed:
Merge branch 'master' of https://github.com/jkemp814/iptv

BellezaEmporium committed:
Merge branch 'master' into master

jkemp814 committed:
Added a ton of logos

jkemp814 committed:
Merge branch 'master' of https://github.com/jkemp814/iptv

jkemp814 committed:
Merge branch 'iptv-org:master' into master

jkemp814 committed:
Apparently i.imgur stop adding file extensions. Had to repair

jkemp814 committed:
Fixed quotes

jkemp814 committed:
Fixed clerical error

jkemp814 committed:
Matched group-title for Circle

jkemp814 committed:
Quotes!

jkemp814 committed:
Merge branch 'iptv-org:master' into master

Dum4G committed:
Merge branch 'master' into master

jkemp814 committed:
Few more edits

Dum4G commented:
Don't do anything. Follow-up improvements are coming

Dum4G commented:
Line 58: tvg-logo is too big
<s>Line 206-208: tvg-lang is Spanish</s>
Line 264-266: tvg-logo is too big
<s>Line 377: isn't it the same channel as 375?, Relax genre</s>
Line 393: tvg-logo must be https://upload.wikimedia.org/wikipedia/commons/2/2b/Black_Jesus_title_card.png then</s>
Line 395 - 413 : tvg-logo is too big
Line 417  : tvg-logo is too big
<s>Line 553-559: tvg-id contains special letter</s>
<s>Line 605-611: tvg-id doesnt't have country postfix</s>
Line 647-649  : tvg-logo is too big
Line 1165  : tvg-logo is too big
Line 1387  : tvg-logo is too big
Line 1505  : tvg-logo is too big
Line 1549  : tvg-logo is too big
In general od.lk... tvg-logos are too big
Line 2061  : tvg-logo is too big
Line 2347  : tvg-logo is too big
Line 2355  : tvg-logo is too big
Line 2455-2459  : tvg-logo is too big

I've gone through 4300 of 6700 lines

Dum4G commented:
Now push this to your branch
https://gist.githubusercontent.com/Dum4G/92c627b5dae6468d92588b10644e5af5/raw/faa82e6530b7f268f84cb360b1d40b90b763e061/us.m3u

jkemp814 commented:
I'm new to this Dum4G. How do I do that?

Dum4G commented:
Via Add a file in a channel folder (call it us.m3u) or any other mean that you use to push your changes

jkemp814 committed:
Add files via upload

jkemp814 committed:
Delete us.m3u

jkemp814 commented:
Okay, I added.

Dum4G commented:
I see you have added and removed the us file

jkemp814 commented:
I switched branches, so I deleted it from my master branch. I hadn't changed to the channel's folder. Furthermore, I was in the root of IPTV. So I switched to channel folder and added/uploaded.

Dum4G commented:
As of now changes from the master branch are the only ones that Github fetches into this pull request

jkemp814 commented:
Okay, I will switch back and do it from there. I just wanted to test it on another branch to see if it's works. One moment.

jkemp814 committed:
Add files via upload

Dum4G commented:
One more time please
https://gist.githubusercontent.com/Dum4G/f1e6b270e536837cdc58f754b0567e1b/raw/70a21d54eabbc3946ddada2515f9cf6a374f08e9/us.m3u

Dum4G commented:
One more time, please
https://gist.githubusercontent.com/Dum4G/f57e95bb578087de8824024a5b924492/raw/99041ce0d8f6938cecffc092aae05ef667779952/us.m3u

jkemp814 committed:
Add files via upload

Dum4G commented:
https://github.com/iptv-org/iptv/pull/3521#issuecomment-880117024

So here is my final review list. If you don't willing to fix this in this PR I can approve it

jkemp814 commented:
I'm not going to make any changes, but I seen you edited 2/47 at line 5306. tvg-name="Revry Queer 2/47" is still the same.

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -6137,27 +6137,27 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxvanityfair/CDN/playlist.m3u
+#EXTINF:-1 tvg-id="VENN.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/VENN_552x552.png?raw=true" group-title="Entertainment",VENN
 https://venntv-localnow.amagi.tv/index.m3u8
-#EXTINF:-1 tvg-id="c648ce" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/VENN_552x552.png?raw=true" group-title="Sports",VENN
+#EXTINF:-1 tvg-id="c648ce" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/VENN_552x552.png?raw=true" group-title="Entertainment",VENN
```
TVG-id does not match tvg-name

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -5303,7 +5303,7 @@ https://linear-44.frequency.stream/dist/localnow/44/hls/master/playlist.m3u8
 #EXTINF:-1 tvg-id="RevryNews.us" tvg-name="Revry News" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/AdIThwq.png" group-title="Lifestyle",Revry News
 https://revry-revrynews-1-eu.rakuten.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="RevryQueer247.us" tvg-name="Revry Queer 2/47" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5fc81016d98cab623846a4f3" group-title="Lifestyle",Revry Queer 2/47
+#EXTINF:-1 tvg-id="RevryQueer247.us" tvg-name="Revry Queer 2/47" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5fc81016d98cab623846a4f3" group-title="Lifestyle",Revry Queer 24/7
```
2/47 ?

jkemp814 replied:
I took it off completely, suggested by Dum4G.

Dum4G commented:
> at line 5306
Uh, can you fix this before approval? In fact, we can remove 24/7 entirely

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -6137,27 +6137,27 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxvanityfair/CDN/playlist.m3u
+#EXTINF:-1 tvg-id="VENN.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/VENN_552x552.png?raw=true" group-title="Entertainment",VENN
 https://venntv-plex.amagi.tv/hls/amagi_hls_data_venntvAAA-venntv-plex/CDN/playlist.m3u8?us_privacy=1---&did=df8e1a36-847d-5096-86a7-3803ed330ede&dnt=0&X-Plex-Token=MorUy57ijWhGe4ixZb_T&channelId=5f74cf7dffbe690040689f3b%20
-#EXTINF:-1 tvg-id="venntv.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/VENN_552x552.png?raw=true" group-title="Sports",VENN
+#EXTINF:-1 tvg-id="venntv.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/VENN_552x552.png?raw=true" group-title="Entertainment",VENN
```
Inconsistencies with tags above

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -6137,27 +6137,27 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxvanityfair/CDN/playlist.m3u
+#EXTINF:-1 tvg-id="VENN.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/TheRokuChannel_880x880.png?raw=true" group-title="Entertainment",VENN
 https://venntv-roku.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="VennTV.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/VENN_552x552.png?raw=true" group-title="Sports",VENN (1080p)
+#EXTINF:-1 tvg-id="VennTV.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/VENN_552x552.png?raw=true" group-title="Entertainment",VENN (1080p)
```
Inconsistencies with tags above

LaneSh4d0w commented on channels/us.m3u:
```diff
@@ -6137,27 +6137,27 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxvanityfair/CDN/playlist.m3u
+#EXTINF:-1 tvg-id="VennTV.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/VENN_552x552.png?raw=true" group-title="Entertainment",VENN (1080p)
 https://venntv-samsungus.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="VennTV.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/VENN_552x552.png?raw=true" group-title="Sports",VENN
+#EXTINF:-1 tvg-id="VennTV.us" tvg-name="VENN" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/VENN_552x552.png?raw=true" group-title="Entertainment",VENN
```
Inconsistencies with tags above

jkemp814 committed:
Add files via upload

jkemp814 committed:
Completed Oversized logos

jkemp814 commented:
okay I may have a conflit. I had made all those changes before saving my file. I overwrote the newer file from your link. If I do it again will it infact overwrite my changes doing it this way?

jkemp814 commented:
I think we will be okay because I have did a pull before making my edits, so overwriting it sould still have both our changes

jkemp814 committed:
Made edits per LaneSh4d0w

Dum4G commented:
Latest 3 commits basically revert everything I fixed from my end

jkemp814 commented:
Well, I can push your links again and redo my edits. I have the logos on i.imgur

jkemp814 commented:
If I did a pull on my end, after pushing your changes, wouldn't that mean I was working with your changes?

Dum4G commented:
When you did Add files via upload commit it was probably an old file

jkemp814 commented:
You can see your changes reverted?

Dum4G commented:
You can click on commit names and see what's changed

jkemp814 commented:
Okay, so I'm clear, if I go back and push those two files again, what would that do to the commits already in the repository?

Dum4G commented:
https://gist.githubusercontent.com/Dum4G/f57e95bb578087de8824024a5b924492/raw/99041ce0d8f6938cecffc092aae05ef667779952/us.m3u

If you post this file it will reinstate my changes and revert yours. You can modify this file to include your changes

jkemp814 commented:
Are we talking about the lines you struck out in that list you posted? Or were there other edits in the us.m3u outside this pull-request?

Dum4G commented:
I did some slight tweaks to existing lines and nothing outside this PR

LaneSh4d0w commented:
Anything related to cleaning or "tweaking" links, it'll be either I or Dum doing it in another PR.

jkemp814 commented:
> Anything related to cleaning or "tweaking" links, it'll be either I or Dum doing it in another PR

So let everything stand as it is?

Or go fowarded reverting my changes (180 lines) by pusing that link posted and redoing them? I'm a little confused.

Dum4G commented:
It's better to redo them as I also did plenty of changes here and there

LaneSh4d0w commented:
> 
> 
> > Anything related to cleaning or "tweaking" links, it'll be either I or Dum doing it in another PR
> 
> So let everything stand as it is?
> 
> Or go fowarded reverting my changes (180 lines) by pusing that link posted and redoing them? I'm a little confused.

Make your changes from the file Dum4G provided you, and once you've finished with that, push the freshly modified link.

jkemp814 commented:
I think his changes are still there. I did a pull to my local branch  before I started my edits. Then once completed I thought when I seen what I thought was a new link to push, (I was working near his list of changes in his post I did that push again. Ten saved my file locally and it prompt me to 'compare or 'overwrite' I chose to overwrite thinking all changes were already in my local branch, plus my changes. So that should be whats in the repository now. I'm I asuming correctly here?

jkemp814 committed:
Add files via upload

Dum4G commented:
You can now see what changes you did and where as well as how many of other editions were reverted before
https://github.com/iptv-org/iptv/pull/3521/commits/f53fc52b012449197255f111ca5408ddbadae173

Dum4G committed:
Merge branch 'master' into master

jkemp814 committed:
Redo complete

jkemp814 committed:
Merge branch 'master' of https://github.com/jkemp814/iptv

jkemp814 committed:
Corrected URL

jkemp814 commented:
Roku:     880x880      (68) These are huge! When I view them, they are as big as they are described in the URL.
Redbox:     512x512      (54) They are 512x512
   Xumo:     456x456      (20)  Actual size
 STIRR:     1920x1080  (175)  Obviously they were converted down. Very tiny when you view them. (Stingray)
